languages/go/exercises/concept/maps/.docs/introduction.md

```diff
@@ -0,0 +1,58 @@
+In go, `map` is a built-in data type that represent a key/value store or hash table. In other programming language, you might know `map` as `dict` or associative array. If you're not familiar with such concept, map you can think `map` like a dictionary, which every word is the `key` and the definition is the `element` of `map`.
```

Comment:
I feel like you've done a good job of explaining what a map is, and don't need the last line:

```suggestion
In go, `map` is a built-in data type that represent a key/value store or hash table. In other programming language, you might know `map` as `dict` or associative array.
```

Reply:
Are you sure? I think that line is needed, I know Jeremy said the introduction is for people coming from other language, meaning they have prior coding experience but just to put everyone on the same playing field

Reply:
Maybe reorder slightly to define it more up front, and move key/value and hash into the examples? e.g.

```suggestion
In go, `map` is a built-in data type that maps keys to values. In other programming language, you might be familiar with the concept of `map` as a dictionary, hash table, key/value store or an associative array.
```

(Using this as my reference: https://tour.golang.org/moretypes/19)